metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-env.xml

```diff
@@ -77,132 +71,17 @@
-            <hidden>METRON_CLIENT</hidden>
-        </value-attributes>
-        <on-ambari-upgrade add="true"/>
-    </property>
-    <property>
-        <name>metron_jdbc_platform</name>
-        <value></value>
-        <description>Database platform used by Metron.  One of: hsqldb, h2, oracle, mysql, postgresql</description>
-        <display-name>Metron JDBC platform</display-name>
-    </property>
-    <property>
-        <name>metron_jdbc_client_path</name>
-        <description>Path to JDBC jar for selected platform</description>
-        <display-name>Metron JDBC client path</display-name>
-        <value/>
-        <value-attributes>
-            <empty-value-valid>true</empty-value-valid>
-        </value-attributes>
-    </property>
-    <property>
-        <name>metron_temp_grok_path</name>
-        <description>Temporary local file path where grok patterns are written during testing</description>
-        <value>./patterns/temp</value>
-        <display-name>Metron temp grok path</display-name>
-    </property>
-    <property>
-        <name>metron_default_grok_path</name>
-        <description>Default HDFS directory path used when storing Grok patterns</description>
-        <value>/apps/metron/patterns</value>
-        <display-name>Metron default grok path</display-name>
-    </property>
-    <property>
-        <name>metron_spring_options</name>
-        <description>Additional Spring options not included in the rest_application.yml file</description>
-        <display-name>Metron Spring options</display-name>
-        <value/>
-        <value-attributes>
-            <empty-value-valid>true</empty-value-valid>
-        </value-attributes>
-    </property>
-    <property>
         <name>metron_topic_retention</name>
         <description>Kafka Retention in GB</description>
         <value>10</value>
         <display-name>Topic Retention</display-name>
     </property>
     <property>
-        <name>parsers</name>
-        <value>bro,snort,yaf</value>
-        <description>Metron parsers to deploy</description>
-        <display-name>Metron Parsers</display-name>
-    </property>
-    <property>
-        <name>metron_indexing_topology</name>
-        <value>indexing</value>
-        <description>The Storm topology name for Indexing</description>
-        <display-name>Indexing Topology Name</display-name>
-    </property>
-    <property>
         <name>es_cluster_name</name>
         <value>metron</value>
         <description>Name of Elasticsearch Cluster</description>
         <display-name>Elasticsearch Cluster Name</display-name>
     </property>
-    <property>
-        <name>geoip_url</name>
-        <value>http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz</value>
-        <description>Location of the GeoIP data to load.</description>
-        <display-name>GEOIP Load Datafile URL</display-name>
-    </property>
     <property require-input="true">
         <name>es_hosts</name>
```

Comment:
I left this property in metron-env.xml (our default top-level env file) because it should rarely ever change.  I can create a separate component for it if you think that would be helpful.